dubbo-common/src/main/java/org/apache/dubbo/common/metrics/model/MethodMetric.java

```diff
@@ -0,0 +1,131 @@
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dubbo.common.metrics.model;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import static org.apache.dubbo.common.constants.MetricsConstants.TAG_APPLICATION_NAME;
+import static org.apache.dubbo.common.constants.MetricsConstants.TAG_GROUP_KEY;
+import static org.apache.dubbo.common.constants.MetricsConstants.TAG_HOST;
+import static org.apache.dubbo.common.constants.MetricsConstants.TAG_INTERFACE_KEY;
+import static org.apache.dubbo.common.constants.MetricsConstants.TAG_METHOD_KEY;
+import static org.apache.dubbo.common.constants.MetricsConstants.TAG_PARAMETER_TYPES_DESC;
+import static org.apache.dubbo.common.constants.MetricsConstants.TAG_VERSION_KEY;
+import static org.apache.dubbo.common.utils.NetUtils.getLocalHost;
+
+public class MethodMetric {
+    private String applicationName;
+    private String interfaceName;
+    private String methodName;
+    private String parameterTypesDesc;
+    private String group;
+    private String version;
+
+    public MethodMetric() {
+
+    }
+
+    public MethodMetric(String applicationName, String interfaceName, String methodName, String parameterTypesDesc, String group, String version) {
+        this.applicationName = applicationName;
+        this.interfaceName = interfaceName;
+        this.methodName = methodName;
+        this.parameterTypesDesc = parameterTypesDesc;
+        this.group = group;
+        this.version = version;
+    }
+
+    public String getInterfaceName() {
+        return interfaceName;
+    }
+
+    public void setInterfaceName(String interfaceName) {
+        this.interfaceName = interfaceName;
+    }
+
+    public String getMethodName() {
+        return methodName;
+    }
+
+    public void setMethodName(String methodName) {
+        this.methodName = methodName;
+    }
+
+    public String getParameterTypesDesc() {
+        return parameterTypesDesc;
+    }
+
+    public void setParameterTypesDesc(String parameterTypesDesc) {
+        this.parameterTypesDesc = parameterTypesDesc;
+    }
+
+    public String getGroup() {
+        return group;
+    }
+
+    public void setGroup(String group) {
+        this.group = group;
+    }
+
+    public String getVersion() {
+        return version;
+    }
+
+    public void setVersion(String version) {
+        this.version = version;
+    }
+
+    public Map<String, String> getTags() {
+        Map<String, String> tags = new HashMap<>();
+        tags.put(TAG_HOST, getLocalHost());
```

Comment:
It's better to add tag both for IP address and hostname. In the k8s cluster, the node name uses hostname.